tests/python/pytest_opinfos.py

```diff
@@ -1103,6 +1105,33 @@ def torch_reshape_sym_fn(input_tensor, output_shaped_tensor):
 )
 tensor_creation_ops.append(uniform_opinfo)
 
+matmul_ops = []
+
+
+def matmul_wrapper(fd, data_type, a, b):
```

Comment:
We should have a quick comment indicating why we need separate OpInfo definitions for `fp16` and `bf16`.